lamb/src/main/java/com/scalar/kelpie/lamb/LambProcessor.java

```diff
@@ -0,0 +1,42 @@
+package com.scalar.kelpie.lamb;
+
+import com.scalar.dl.client.service.ClientService;
+import com.scalar.kelpie.config.Config;
+import com.scalar.kelpie.modules.TimeBasedProcessor;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import javax.json.JsonObject;
+
+public class LambProcessor extends TimeBasedProcessor {
+  private static final String TEST_CONFIG_TABLE = "benchmark_config";
+  private static final String CONTRACTS = "target_contract";
+
+  private final ClientService service;
+  private final String contractName;
+  private final ArgumentBuilder argumentBuilder;
+
+  public LambProcessor(Config config) throws IOException, FileNotFoundException {
```

Comment:
FileNotFoundException is not needed because the same reason in the above comment.